tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
             cy.checkFrameNum(groundTruthFrames[3]);
         });
     });
+
+    describe('Testing case ground truth job list', () => {
+        const imagesCount = 60;
+        const imageFileName = 'ground_truth_2';
+        const width = 100;
+        const height = 100;
+        const posX = 10;
+        const posY = 10;
+        const color = 'gray';
+        const archiveName = `${imageFileName}.zip`;
+        const archivePath = `cypress/fixtures/${archiveName}`;
+        const imagesFolder = `cypress/fixtures/${imageFileName}`;
+        const directoryToArchive = imagesFolder;
+
+        before(() => {
+            cy.visit('/tasks');
+            cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+            cy.createZipArchive(directoryToArchive, archivePath);
+            cy.createAnnotationTask(
+                taskName, labelName, attrName,
+                textDefaultValue, archiveName, false,
+                { multiJobs: true, segmentSize: 2 },
+            );
+            cy.openTask(taskName);
+            cy.url().then((url) => {
+                taskID = Number(url.split('/').slice(-1)[0].split('?')[0]);
+            });
+            cy.get('.cvat-job-item').first().invoke('attr', 'data-row-id').then((val) => {
+                jobID = val;
+            });
+        });
+
+        it('Create ground truth job, compute quality report, check jobs table', () => {
+            cy.createJob({
+                ...jobOptions,
+                quantity: 100,
+            });
+
+            cy.createRectangle(rectangleTrack);
+            cy.saveJob();
+
+            cy.interactMenu('Finish the job');
+            cy.get('.cvat-modal-content-finish-job').within(() => {
+                cy.contains('button', 'Continue').click();
+            });
+
+            createTaskQualityReport(taskID);
+
+            for (let i = 0; i < 3; i++) {
```

Comment:
3 is a magic number, it is not obvious that it result of dividing number of images to segment size